### PR TITLE
hey let me plug my social media into a mod pack 25k people use becaus…

### PR DESCRIPTION
**Pull request by MicrocontrollersDev**
…e haha funny

microcontrollers

### CODE REVIEW

**Comment by MicrocontrollersDev** [comment]
LGTM ready for merge

**Review by koxx12-dev-is-bad — APPROVED** [review]
so true lgtm

File: docs/assets/funfacts.txt [hunk 1]
```diff
@@ -31,6 +31,7 @@ Try to do something creative! It could be music, art, coding...\nYou won't know
 https://howoldisminecraft189.today/
 SkyClient is for more than Skyblock, even though it started out that way.
 SkyClient is a zero profit organization; we have unpaid volunteers around the globe.
+Microcontrollers
```
Comment (KTibow) [inline docs/assets/funfacts.txt:34]:
```suggestion
One great way to get started with coding\nis with a microcontroller, a tiny computer.\nhttps://www.arduino.cc/
```

Comment (RayDeeUx) [reply docs/assets/funfacts.txt:34]: based

**Pull request merged**